tests/test_dimension.py

```diff
@@ -566,14 +566,14 @@ def test_subsampled_fd(self):
         op.apply(time_M=0, x_M=11, y_M=11)
         # Verify that u2 contains subsampled fd values
         assert np.all(u2.data[0, :, :] == 2.)
-        assert np.all(u2.data[1, 0, 0] == 20.)
-        assert np.all(u2.data[1, -1, -1] == -20.)
-        assert np.all(u2.data[1, 0, -1] == 0.)
-        assert np.all(u2.data[1, -1, 0] == -0.)
-        assert np.all(u2.data[1, 1:-1, 0] == 10.)
-        assert np.all(u2.data[1, 0, 1:-1] == 10.)
-        assert np.all(u2.data[1, 1:-1, -1] == -10.)
-        assert np.all(u2.data[1, -1, 1:-1] == -10.)
+        assert np.all(u2.data[1, 0, 0] == 0.)
+        assert np.all(u2.data[1, -1, -1] == -40.)
```

Comment:
why are these changing

Reply:
changes in FD, was [-.5, 0, -5] previously, is [0, -1, 1] now (for that order)